### PR TITLE
steps-jira-post-build-details 0.0.1

### PR DESCRIPTION
**Pull request by stanrud**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by vshah23** [comment]
https://bitrise.io/integrations/steps/post-jira-comment-with-build-details

This step seems to provide similar functionality to the one you have created. Does it already satisfy your needs or are you able to contribute a feature enhancement to it instead?

File: steps/steps-jira-post-build-details/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+source:
+  git: https://gitlab.com/rudnicst/steps-post-jira-comment-with-build-detail.git
+  commit: 43876f2fd347c1640a4772fc571efd249e1e6ca5
+type_tags:
+- notification
+toolkit:
+  go:
+    package_name: gitlab.com/rudnicst/steps-post-jira-comment-with-build-detail
+is_requires_admin_user: false
+is_always_run: false
```
Comment (vshah23) [inline steps/steps-jira-post-build-details/0.0.1/step.yml:18]: Since this is a `notification` type step `is_always_run` must be true.

**Review by vshah23 — CHANGES_REQUESTED** [review]
Step’s ID should not start with `steps-`.

**Comment by godrei** [comment]
I'm closing this PR because of the inactivity.

**Pull request closed without merging**